NuRadioReco/utilities/dataservers.py

```diff
@@ -0,0 +1,115 @@
+import requests
+import os
+import filelock
+import logging
+import shutil
+from glob import glob
+
+logger = logging.getLogger('NuRadioReco.dataservers')
+
+dataservers = ["https://rnog-data.zeuthen.desy.de", "https://rno-g.uchicago.edu/data/desy-mirror"]
+
+def get_available_dataservers_by_responsetime(dataservers=dataservers):
+    """ requests a small index file from the list of dataservers and returns a list of responsive ones ordered by elapsed time """
+    response_times = []
+    available_dataservers = []
+
+    for dataserver in dataservers:
```

Comment:
it could make sense to parallelize this, though in practice it probably doesn't matter very much with just two servers...

Reply:
yeah... right.. I don't expect that we will ever have >> 2 servers. I suggest to leave it as is.